build-deck.sh

```diff
@@ -2,6 +2,6 @@
 
 cd deck
 cat collection-dump.sql | sqlite3 collection.anki2
-find  -type f ! -name "collection-dump.sql" | xargs zip "../Language IPA.apkg"
+find  -type f ! -name "collection-dump.sql" | xargs zip "../IPA-anki-deck.apkg"
```

Comment:
I would like to keep `Language IPS.apkg` in order to be consistent with the other decks that have been created. Why is this change necessary?
Possibly in the README should be written that `Language` should be replaced with the target language here and in the gitlab action.